custom_model_runner/datarobot_drum/drum/gpu_predictors/__init__.py

```diff
@@ -240,3 +249,51 @@ def download_and_serve_model(self, openai_process: DrumServerProcess):
 
     def terminate(self):
         raise NotImplementedError
+
+
+class MLOpsStatusReporter:
+    def __init__(
+        self,
+        mlops_service_url,
+        mlops_api_token,
+        deployment_id,
```

Comment:
When this code runs as part of a Custom Model Test, there will be no `deployment_id` so the code needs to support this flow. The _easiest_ is probably to just have this class silently drop the message (or at least right now it looks like it will log a warning which will be noisy)?

Reply:
added the change
https://github.com/datarobot/datarobot-user-models/blob/b978bde13f8362d197835f0c42acaaa05e8f055d/custom_model_runner/datarobot_drum/drum/gpu_predictors/__init__.py#L33-L36